internal/pkg/service/buffer/storage/local/writer/base/base.go

```diff
@@ -36,6 +54,12 @@ func (w *Writer) SliceKey() storage.SliceKey {
 	return w.slice.SliceKey
 }
 
+func (w *Writer) Columns() column.Columns {
+	out := make(column.Columns, len(w.slice.Columns))
+	copy(out, w.slice.Columns)
+	return out
+}
+
```

Comment:
Pridany getter `Columns`, ktory sa pouzije neskor.
Slice sa prenasaju cez referenciu, takze tu sa robi kopia aby to bolo immutable.